app/dashboard/migrations/0091_auto_20180712_1800.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.0.7 on 2018-07-12 18:00
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('dashboard', '0090_auto_20180709_1525'),
+    ]
+
+    operations = [
```

Comment:
i wonder if this should just be a json field with a list of many tx hashes... @mbeacom what do you think?